messaged.c

```diff
@@ -184,16 +186,47 @@ send_message(struct context *ctx, struct contact *con)
 		buf[size - 1] = '\0';
 		size--;
 	}
-	msg_send(ctx, buf, con->name);
+	/* These characters must be escaped. */
+	if (NULL != strchr(buf, '<') || NULL != strchr(buf, '&')) {
```

Comment:
Use `strspn(3)` instead of `strchr(3)`. Thus, you just have one loop to search both characters.

Reply:
I did this (and compared against length), but now I get a compiler warning, but otherwise it's okay.